src/common/analytics/Analytics.ts

```diff
@@ -0,0 +1,53 @@
+export const GA_MEASUREMENT_ID = `G-13GYPPVD4Y`;
+
+type GaActionDefault = "page_view";
+type GaAction =
+  | "magic_demo_start"
+  | "magic_demo_issue"
+  | "magic_demo_downloaded"
+  | "magic_demo_file_drop"
+  | "magic_demo_drop_off";
+type GaCategory = "magic_demo";
+
+interface GaEventProps {
+  action: GaAction;
+  category: GaCategory;
+  label?: string;
+  value?: number;
+}
+
+interface GaPageViewProps {
+  action: GaActionDefault;
+}
+
+export const validatePageViewEvent = (event: GaPageViewProps): void => {
+  const { action } = event;
+  if (!action) console.error("Action is required");
+  if (action && typeof action !== "string") console.error("Action must be a string");
```

Comment:
actually thinking about this validator, is it required?
i mean, so far every call to gaPageView is determined by us, there isnt anyway to call it from the user's point of view right?

Reply:
as discussed, we will ignore this for now since opencerts has this kind of behavior